### DISCUSSION TITLE
Please How Can I reduce the size of each emoji in NotoColorEmoji.ttf

### DISCUSSION THREAD

**Discussion by Asalerm** [fonttools/fonttools#3592] [Q&A]
All I am trying is to reduce the size of each emoji in the .ttf file and make it permanantly 10% of it original size. Please I need help.

**Comment by justvanrossum** [comment]
One way would be to set the unitPerEm field to 10x its original value. Another would be to use the scaleUpem tool to scale down, then set the unitsPerEm field back to the original.

**Reply by Asalerm** [reply]
Please can you help me do this and show how to do it, I tried but no success,

**Reply by behdad** [reply]
> One way would be to set the unitPerEm field to 10x its original value. Another would be to use the scaleUpem tool to scale down, then set the unitsPerEm field back to the original.

The scaleUpem tool does not resize PNGs, which is what the requester is asking for I think.

**Reply by behdad** [reply]
> I tried but no success,

What *did* you try?

Try dumping the font with the `ttx` command-line argument that creates the PNGs as separate files, then modify those, and see what else you need to change in the font...

**Reply by Asalerm** [reply]
Yes I changed the NotoColorEmoji-emojicompat.ttf to NotoColorEmoji-emojicompat.ttx and when I edit the NotoColorEmoji-emojicompat.ttx and change the unitsPerEm nothing happens.

**Reply by Asalerm** [reply]
I use this command to create the .ttx file  (ttx -z extfile NotoColorEmoji-emojicompat.ttf)  after it created the .ttx I edit it and save it and run this command to re-create the .ttf file again (ttx Modified_NotoColorEmoji.ttx).

**Reply by behdad** [reply]
Tell us again, what are you trying to achieve?

**Reply by behdad** [reply]
So you don't need the file size to be smaller; just the rendering?

**Reply by Asalerm** [reply]
What I want to achieve is this, emojis of NotoColorEmoji.ttf appears too big, I want to reduce their sizes to 10% of their original size. I dont care much about the file size. I just need the emoji rendering to be very smaller in size when rendering.

**Reply by Asalerm** [reply]
> So you don't need the file size to be smaller; just the rendering?

Yes only the rendering to be very small.

**Reply by Asalerm** [reply]
Hi any suggestions please!!
@behdad

**Reply by behdad** [reply]
I'm fairly certain that your problem is not the embedded emoji size, but that your graphics library is NOT scaling them to the desired size.

At any rate, I can't spend time figuring this out for you. If you are willing to pay, I'm sure there are people who can help you figure it out.
